consensus/hotstuff/validator/validator.go

```diff
@@ -144,20 +144,16 @@ func (v *Validator) ValidateVote(vote *model.Vote, block *model.Block) (*flow.Id
 	}
 
 	// check whether the signature data is valid for the vote in the hotstuff context
-	valid, err := v.verifier.VerifyVote(voter, vote.SigData, block)
+	err = v.verifier.VerifyVote(voter, vote.SigData, block)
 	if err != nil {
 		switch {
 		case errors.Is(err, signature.ErrInvalidFormat):
-			return nil, newInvalidVoteError(vote, err)
-		case errors.Is(err, model.ErrInvalidSigner):
```

Comment:
`VerifyVote` will not return model.ErrInvalidSigner error, because it assumes the caller has already verified the signer, which [we did it here](https://github.com/onflow/flow-go/pull/1741/files#diff-8e933e078b2e6031be4617d76cfe25046e38ac012720b757b382869e6a509aeaL140).